### DISCUSSION TITLE
autocomplete multiple hitting enter doesn't select the item that is an exact match

### DISCUSSION THREAD

**Discussion by runxc1** [primefaces/community#3008] [PrimeVue]
When using the autocomplete with no item highlighted when the user has typed the exact label text of an item and hit's enter the user is expecting that the item they just typed in be selected.    This feels like something that could be handled internally as the enter only selects the item highlighted currently but intuitively if you have already typed out an entry and hit enter you want it to be added.  Most tagging systems work this way.

**Comment by j0rgedev** [comment]
Hello @runxc1. Could you please provide a reproduction? I'll be happy to look into it.
